dev/create-release/release-build.sh

```diff
@@ -258,6 +258,7 @@ if [[ "$1" == "package" ]]; then
   LFTP mkdir -p $dest_dir
   LFTP mput -O $dest_dir 'spark-*'
   LFTP mput -O $dest_dir 'pyspark-*'
+  LFTP mput -O $dest_dir 'SparkR-*'
```

Comment:
Not sure - Ah - I see whats going on. We create the special directory called latest and copy stuff into that first. I always assumed that was a symlink. Hmm unfortunately I merged this - I can create a new PR. Ugh